Volume/Views/BrowserView.swift

```diff
@@ -14,11 +15,21 @@ struct BrowserView: View {
     @EnvironmentObject private var userData: UserData
 
     let article: Article
+    let entryPoint: EntryPoint
+
     var isShoutoutsButtonEnabled: Bool {
         userData.canIncrementShoutouts(article)
     }
 
+    private var params: [String: Any] {
+        Parameters.params(for: .article, id: article.id, at: entryPoint)
+    }
+
     private func incrementShoutouts() {
+        if isShoutoutsButtonEnabled {
```

Comment:
You don't need this check. The function can only be called if the button is enabled. Also, it doesn't make sense to only put the event in here but not the actual incrementing network request.